emails/templates/missed-reveal.js

```diff
@@ -0,0 +1,54 @@
+const {
+  addressBadge,
+  action,
+  base,
+  infobox,
+  link,
+} = require('../template-utils')
+const { accountData } = require('../mock-utils')
+
+module.exports = function() {
+  return {
+    template: base(
+      {
+        title: 'Notifications',
+        subtitle: `Your account ${addressBadge()} received a notification on {{date}}`,
+      },
+      `
+        ${infobox({
+          mode: 'negative',
+          primary: `Your vote wasn’t revealed on time`,
+          secondary: `Some of your ${link(
+            'locked ANJ balance',
+            '{{lockedAnjBalanceUrl}}'
+          )} has been forfeit.`,
+        })}
+        ${action('Learn more', '{{learnMoreUrl}}', { padding: '16px 0 0' })}
+      `
+    ),
+    templateText: `
+      Aragon Court Notifications
+
+      Your account {{account}} received a notification on {{date}}:
+
+      Your vote wasn’t revealed on time. Some of your locked ANJ [1] balance has
+      been forfeit.
+
+      Learn more: {{learnMoreUrl}}
+
+      This service is provided by Aragon One AG [2]. You are receiving this
+      email because you are subscribed to Aragon Court Email Notifications. You
+      can contact us at support@aragon.org if you not longer wish to receive
+      these.
```

Comment:
I guess we could add some helper functions for these parts to avoid repetition in every template

Reply:
Yes... maybe. The `[2]` part is a bit annoying here to extract this as a helper.